src/Collect/Support/HigherOrderTapProxy.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Does anything rely on this? Is this providing anything that we don't have in the current version?

Reply:
Nope. This is used by the `tap()` helper we don't have here. I just removed it.

Reply:
Thanks!